addons/udes_stock/models/stock_picking.py

```diff
@@ -1244,17 +1244,23 @@ def unlink_empty(self):
     def _check_entire_pack(self):
         """
             Override to avoid values at result_package_id when
-            user scans products
+            user scans products/avoid _check_entire_pack onf products
         """
         pickings = self.filtered(
             lambda p: p.picking_type_id.u_user_scans != 'product' and
                       p.picking_type_id.u_target_storage_format != 'product')
         super(StockPicking, pickings)._check_entire_pack()
-        package_move_lines = pickings \
-            .filtered(lambda p: p.picking_type_id.u_target_storage_format == 'package') \
-            .mapped("move_line_ids")
-        if package_move_lines:
-            package_move_lines.write({'u_result_parent_package_id': False})
+
+    def _set_u_result_parent_package_id(self):
+        """ Override function to avoid setting result_package_id when user
+        the storage format is package/pallet of products"""
+        package_pickings = self \
+            .filtered(
+                lambda p: p.picking_type_id.u_target_storage_format in
+                ('package' or 'pallet_packages')
```

Comment:
('package', 'pallet_packages')

Reply:
In fact, this should be pallet products not pallet packages

Reply:
Also we don't want to set it for products. So probably do it the other way around:
`pallet_package_pickings = XXX`
and use it when calling super.